Update multicore-jobs.md (#44)
OSGConnect/connectbook: start/resources/multicore-jobs.md

[TOC]

Please note, the OSG has limited support for multicore jobs. Multicore jobs
can be submitted for threaded or OpenMP applications. To request multiple cores
(aka cpus) use the HTCondor *request_cpus* attribute in your submit file. 

Example:

    request_cpus = 8

> Currently, multicore jobs can request a maximum of 8 cpus. 

Reviewer: @rynge is this true?

Reviewer 2: We can do more than 8, but it is still uncommon to have those resources available at all times.

I think the auto-detection part is important. When you request something like 8, there is a good chance it is on a larger machine, and if you let the code auto detect it will start more threads than the 8 requested.

Reviewer 2: Maybe change the language to say "we recommend a maximum of 8"

> Currently, multicore jobs can request a maximum of 8 cpus. 

Reviewer: ```suggestion
We recommend requesting a maximum of 8 cpus. 
```
